### PR TITLE
[Don't merge] New resource types added for Towboat private preview 

### PR DESCRIPTION
**Pull request by assing**
Initial changes 

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

**Review by KrisBash — APPROVED** [review]
LGTM

**Comment by anuchandy** [comment]
@assing I see that swagger is targeting private preview features. Are we planning to generate SDKs for this features and release it PUBLIC?

**Comment by anuchandy** [comment]
@assing Ok, can you share your email so that I can find you in the Teams/Skype?

**Comment by assing** [comment]
@anuchandy  Email Id: assing@microsoft.com

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: Succesfully -> Successfully

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: Succesfully -> Successfully

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: Succesfully -> Successfully

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: GetbyId -> GetById

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: GetbyId -> GetById

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/examples/Alerts_GetbyId.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/examples/Alerts_GetbyId.json]: GetbyId -> GetById

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json
Comment (nschonni) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json]: seperated -> separated

**Review by nschonni — COMMENTED** [review]
Minor typos. There is also "groupby" and "groupedby" that might need to be camel case

**Comment by assing** [comment]
@anuchandy Can you please review these changes so that I can merge? Actually we want to send another PR for same API version.

**Comment by salameer** [comment]
@anuchandy  what's the update on this PR? should it be merged?

**Comment by assing** [comment]
@salameer Now I don't want to merge this PR.

**Comment by salameer** [comment]
@assing what's blocking this merge, are the resource not live yet?  what's the ETA other wise?

**Comment by assing** [comment]
@salameer API is already live. We haven't exposed this API to customers. We created this API for private preview for Toaboat UX. We are not recommeding this API to customers for their use. That's why I don't want to merge.

**Comment by salameer** [comment]
I seeeee, in the future you might want to consider using the private repo or having this feature live in a branch until ready to go :)

Use the aka.ms\openApiPortal

to manage your next swagger it can help direct you :)

**Comment by salameer** [comment]
@assing I'm not quite understanding the situation here, are you planning to ever merge this PR? if not then we will go ahead and close it. we have a policy around keeping PRs open for so long in the Public repo

**Comment by anuchandy** [comment]
@assing closing this PR, please open a new PR when these APIs gets moved from private to public.

**Pull request closed without merging**
